### PR TITLE
Fix test rename

### PR DESCRIPTION
**Pull request by fernandog**
Fixed:  "TypeError: SICKRAGE.info is undefined" in /home/testRename

### CODE REVIEW

File: gui/slick/js/test-rename.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
-$(document).ready(function() {
-    if (SICKRAGE.info.fanArtBackground) {
-        $.backstretch('showPoster/?show=' + $('#showID').attr('value') + '&which=fanart');
-        $('.backstretch').css('opacity', SICKRAGE.info.fanartBackgroundOpacity).fadeIn(500);
+function metaToBool(pyVar) {
+    var meta = $('meta[data-var="' + pyVar + '"]').data('content');
+    if (typeof meta === 'undefined') {
+        console.log(pyVar + ' is empty, did you forget to add this to main.mako?');
```
Comment (fernandog) [inline gui/slick/js/test-rename.js]: @OmgImAlexis ^

Comment (OmgImAlexis) [inline gui/slick/js/test-rename.js]: That's fine.

**Comment by OmgImAlexis** [comment]
Approved!

File: gui/slick/js/test-rename.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
-$(document).ready(function() {
-    if (SICKRAGE.info.fanArtBackground) {
```
Comment (OmgImAlexis) [inline gui/slick/js/test-rename.js]: Oh.. here's the capital.

**Pull request merged**